src/python/grpcio/grpc/aio/_interceptor.py

```diff
@@ -571,7 +571,7 @@ class InterceptedUnaryUnaryCall(_InterceptedUnaryResponseMixin, InterceptedCall,
     _channel: cygrpc.AioChannel
 
     # pylint: disable=too-many-arguments
-    def __init__(self, interceptors: Sequence[UnaryUnaryClientInterceptor],
```

Comment:
This `__init__` is part of our public interface, so we can't change the type. Instead, we'll want to convert the argument to a `list` within the method.